src/core/client/stream/tabs/Comments/ReplyList/ReplyList.tsx

```diff
@@ -79,6 +80,11 @@ const ReplyList: FunctionComponent<ReplyListProps> = (props) => {
                       showConversationLink={!!comment.showConversationLink}
                       onRemoveAnswered={props.onRemoveAnswered}
                       toggleCollapsed={toggleCollapsed}
+                      ancestorID={
+                        props.flattenLastReply && props.indentLevel === 3
```

Comment:
the `indentLevel` stuff feels a bit magic-number-y. maybe extract into a well-named constant or add comments to explain

Reply:
👍